libs/langchain/langchain/retrievers/google_cloud_enterprise_search.py

```diff
@@ -112,14 +118,14 @@ def __init__(self, **data: Any) -> None:
             serving_config=self.serving_config_id,
         )
 
-    def _convert_search_response(
+    def _convert_unstructured_search_response(
```

Comment:
I feel like this could be edited to reuse more code rather than having two separate functions.

Reply:
i looked into this- the changes are plenty to justify the creation of 2. returned fields are quite different